### PR TITLE
fix: #1240 Language: Text of message

### PR DESCRIPTION
**Pull request by timoxa20**
fix “Mentors requests of the way”, buttons “Accept” and “Decline” in dialogue box aren’t translated into Russian, Ukrainian language

### CODE REVIEW

File: mw-webapp/src/dictionary/WayPageContent.json [hunk 1]
```diff
@@ -768,5 +768,22 @@
       "en": "In the comments you can write anything that does not relate to the previous columns. For example, you can share your thoughts about training, or write to your mentor what topic you would like to study further (in more detail, etc.). The mentor can use this column to support the student, suggest additional activities, try to explain how to improve the learning process (based on statistics and other information), etc.",
       "ua": "У коментарях можна писати все, що не стосується попередніх шпальт. Наприклад, ти можеш поділитися своїми думками про навчання, або написати ментору, яку тему хотів би вивчити додатково (детальніше і т.д). Ментор може через це коло підтримати студента, запропонувати додаткові активності, постаратися пояснити, як поліпшити процес навчання (ґрунтуючись на статистиці та іншої інформації) і т.д."
     }
+  },
+  "ComponentMentorRequestsSection": {
+    "MentorRequestsSection": {
+      "ru": "Запрос метора отслеживать путь:",
+      "en": "Mentors requests of this way:",
+      "ua": "Прохання наставників про цей спосіб:"
```
Comment (Ekaterina1994) [inline mw-webapp/src/dictionary/WayPageContent.json:776]:
```suggestion
      "ru": "Запрос от метора отслеживать путь:",
      "en": "Request from a mentor to track the way:",
      "ua": "Прохання від наставника відстежити шлях:"
```

File: mw-webapp/src/dictionary/WayPageContent.json [hunk 2]
```diff
@@ -768,5 +768,22 @@
...
+    },
+    "ButtonAccept": {
+      "ru": "Принять",
+      "en": "Accept",
+      "ua": "Приймати"
+    },
+    "ButtonDecline": {
+      "ru": "Отказаться",
+      "en": "Decline",
+      "ua": "Відмовитися"
```
Comment (Ekaterina1994) [inline mw-webapp/src/dictionary/WayPageContent.json:786]:
```suggestion
      "ru": "Отклонить",
      "en": "Decline",
      "ua": "Відхилити"
```

File: mw-webapp/src/dictionary/WayPageContent.json [hunk 3]
```diff
@@ -768,5 +768,22 @@
       "en": "In the comments you can write anything that does not relate to the previous columns. For example, you can share your thoughts about training, or write to your mentor what topic you would like to study further (in more detail, etc.). The mentor can use this column to support the student, suggest additional activities, try to explain how to improve the learning process (based on statistics and other information), etc.",
       "ua": "У коментарях можна писати все, що не стосується попередніх шпальт. Наприклад, ти можеш поділитися своїми думками про навчання, або написати ментору, яку тему хотів би вивчити додатково (детальніше і т.д). Ментор може через це коло підтримати студента, запропонувати додаткові активності, постаратися пояснити, як поліпшити процес навчання (ґрунтуючись на статистиці та іншої інформації) і т.д."
     }
+  },
+  "ComponentMentorRequestsSection": {
```
Comment (Ekaterina1994) [inline mw-webapp/src/dictionary/WayPageContent.json:772]:
```suggestion
  "mentorRequestsModal": {
```

Comment (Ekaterina1994) [inline mw-webapp/src/dictionary/WayPageContent.json:772]: We don§t need to wrappers delte ComponentMentorRequestsSection or mentorRequestsSection

File: mw-webapp/src/logic/wayPage/MentorRequestsSection.tsx [hunk 4]
```diff
@@ -58,11 +60,14 @@ interface MentorRequestsSectionProps {
  * Section with requests to become Way mentor
  */
 export const MentorRequestsSection = (props: MentorRequestsSectionProps) => {
+  const {language} = languageStore;
+  const mentorsRequestCollection = LanguageService.way.ComponentMentorRequestsSection;
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/wayPage/MentorRequestsSection.tsx:64]: use LanguageService... in the code without creating const

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/dictionary/WayPageContent.json (hunk 1 above)
Comment (timoxa20) [reply mw-webapp/src/dictionary/WayPageContent.json]: Replaced with the suggested option

File: mw-webapp/src/dictionary/WayPageContent.json (hunk 2 above)
Comment (timoxa20) [reply mw-webapp/src/dictionary/WayPageContent.json]: Replaced with the suggested option

File: mw-webapp/src/dictionary/WayPageContent.json (hunk 3 above)
Comment (timoxa20) [reply mw-webapp/src/dictionary/WayPageContent.json]: Replaced with the suggested option

Comment (timoxa20) [reply mw-webapp/src/dictionary/WayPageContent.json]: Этого я не понял

File: mw-webapp/src/logic/wayPage/MentorRequestsSection.tsx (hunk 4 above)
Comment (timoxa20) [reply mw-webapp/src/logic/wayPage/MentorRequestsSection.tsx]: delete const mentorsRequestCollection

File: mw-webapp/src/dictionary/WayPageContent.json [hunk 5]
```diff
@@ -768,5 +768,22 @@
...
+    "ButtonAccept": {
+      "ru": "Принять",
+      "en": "Accept",
+      "ua": "Приймати"
+    },
+    "ButtonDecline": {
+      "ru": "Отклонить",
+      "en": "Decline",
+      "ua": "Відклонити"
+    }
```
Comment (Ekaterina1994) [inline mw-webapp/src/dictionary/WayPageContent.json:787]:
```suggestion
    "mentorRequestsSection": {
      "ru": "Запрос от метора отслеживать путь:",
      "en": "Request from a mentor to track the way:",
      "ua": "Прохання від наставника відстежити шлях:"
    },
    "buttonAccept": {
      "ru": "Принять",
      "en": "Accept",
      "ua": "Приймати"
    },
    "buttonDecline": {
      "ru": "Отклонить",
      "en": "Decline",
      "ua": "Відклонити"
    }
```

File: mw-webapp/src/logic/wayPage/MentorRequestsSection.tsx [hunk 6]
```diff
@@ -58,11 +60,13 @@ interface MentorRequestsSectionProps {
...
  */
 export const MentorRequestsSection = (props: MentorRequestsSectionProps) => {
+  const {language} = languageStore;
+
   return (
     <>
       <Title
         level={HeadingLevel.h3}
-        text="Mentors requests of this way:"
+        text={LanguageService.way.ComponentMentorRequestsSection.MentorRequestsSection[language]}
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/wayPage/MentorRequestsSection.tsx:69]:
```suggestion
        text={LanguageService.way.ComponentMentorRequestsSection.mentorRequestsSection[language]}
```

File: mw-webapp/src/logic/wayPage/MentorRequestsSection.tsx [hunk 7]
```diff
@@ -76,7 +80,7 @@ export const MentorRequestsSection = (props: MentorRequestsSectionProps) => {
             </Link>
             <DialogClose asChild>
               <Button
-                value='Accept'
+                value={LanguageService.way.ComponentMentorRequestsSection.ButtonAccept[language]}
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/wayPage/MentorRequestsSection.tsx:83]:
```suggestion
                value={LanguageService.way.ComponentMentorRequestsSection.buttonAccept[language]}
```

File: mw-webapp/src/logic/wayPage/MentorRequestsSection.tsx [hunk 8]
```diff
@@ -85,7 +89,7 @@ export const MentorRequestsSection = (props: MentorRequestsSectionProps) => {
             </DialogClose>
             <DialogClose asChild>
               <Button
-                value='Decline'
+                value={LanguageService.way.ComponentMentorRequestsSection.ButtonDecline[language]}
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/wayPage/MentorRequestsSection.tsx:92]:
```suggestion
                value={LanguageService.way.ComponentMentorRequestsSection.buttonDecline[language]}
```

**Review by Ekaterina1994 — APPROVED** [review]

**Pull request merged**